kolibri/plugins/coach/assets/src/routes/planLessonsRoutes.js

```diff
@@ -1,4 +1,5 @@
 import store from 'kolibri.coreVue.vuex.store';
+import TechnicalTextBlock from 'kolibri-common/components/AppError/TechnicalTextBlock.vue';
```

Comment:
See the issue description:
> any components that do not currently exist should be created as empty wrapper components with an appropriate name property

We shouldn't be using `TechnicalTextBlock` as a placeholder here, but creating the dummy wrapper components for each route.

If the existing `ResourceSelection` component doesn't sit in here nicely without errors for now, I think we can live with that, because this is on a temporary route.